src/map/merit.cpp

```diff
@@ -26,6 +26,7 @@
 #include "merit.h"
 #include "packets/char_spells.h"
 #include "utils/charutils.h"
+#include "lua/luautils.h"
```

Comment:
Clang wants this reordered to just below charentity,h

Reply:
I have NO idea how you figured that out from the jumbled mess the sanity checks spit back at me ... But I've just pushed a set of fixes.  It should re-run the checks now.

Reply:
Clang usually starts the line it wants you to change with a minus `-` and what it wants you have instead it starts the line with a plus `+`. It actually straight up gives you the line you should use instead, could pretty much copy paste.